modules/migration-adding-cluster-to-cam.adoc

```diff
@@ -40,9 +40,14 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiw
 * *Cluster name*: The cluster name can contain lower-case letters (`a-z`) and numbers (`0-9`). It must not contain spaces or international characters.
 * *URL*: Specify the API server URL, for example, `\https://<www.example.com>:8443`.
 * *Service account token*: Paste the `migration-controller` service account token.
-* *Exposed route host to image registry*: If you are using direct image migration, specify the exposed route to the image registry of the source cluster, for example, `www.example.apps.cluster.com`.
+* *Exposed route host to image registry*: If you are using direct image migration, specify the exposed route to the image registry of the source cluster.
 +
-You can specify a port. The default port is `5000`.
+To obtain the route, run the following command on the source cluster:
++
+[source,terminal]
+----
+$ oc create route passthrough --service=image-registry --port=5000 -n openshift-image-registry
```

Comment:
@apinnick Here we just consider OCP 3.x as source cluster, right?

Reply:
@XinRedhat 
OCP3 as source cluster is the main cluster is the main use case. Should I make sure that this is commented out for OCP 4.x source cluster?